src/components/orgs/Coupon/List.tsx

```diff
@@ -126,6 +142,7 @@ const CouponList: React.FC<ICouponListProps> = ({ supplierId }) => {
           href="/dashboard"
         />
       )}
+      {CommomUserNavigation}

```

Comment:
_Wrong usage: `{CommomUserNavigation}`_

If it's a component: `<NavigationButton />`
If it's a function: `{ renderNavigationButton() }`